packages/components/src/divider/component.tsx

```diff
@@ -53,6 +53,6 @@ function Divider(
  * }
  * ```
  */
-const ConnectedDivider = contextConnect( Divider, 'Divider' );
+export const Divider = contextConnect( UnconnectedDivider, 'Divider' );
```

Comment:
An interesting side effect of adding this named export is that, the `displayName` now becomes whatever this const is named. Since "connected" is an internal concern and Divider isn't even usable when unconnected, I think it would be better to name this `Divider` rather than `ConnectedDivider`.

Reply:
> I thought that the `displayName` was [assigned by `contextConnect`](https://github.com/WordPress/gutenberg/blob/811a2427b93a764206be2cf4ade101edd18d2608/packages/components/src/ui/context/context-connect.ts#L70) based on the namespace passed as the 2nd argument (which is also `Divider` in this case).

It's surprising to me too! It must be some kind of webpack thing...? You can observe the same behavior in React devtools, not just in Storybook code snippets.